### DISCUSSION TITLE
Is the Music style remix discontinued?

### DISCUSSION THREAD

**Discussion by ViktorKuzev** [intel/openvino-plugins-ai-audacity#174] [Q&A] [answered]
I found openvino from a YouTube video showing the "Music style remix" effect. 
So I got audacity and I installed openvino but I can't find it, nor I can find anything on github about it. 
What happened to it?

**Comment by RyanMetcalfeInt8** [comment, marked as answer]
Hi @ViktorKuzev,

Yes, that was one of our initial features that we supported in our v3.4.2-R1 release. It used 'Riffusion' under the hood (stable diffusion for audio). Once we integrated the newer features based on MusicGen LLM (Music Generation / Continuation), we decided to remove support for 'style remix' -- at least until we re-implement some variant of it using the newer MusicGen LLM.

**Reply by ViktorKuzev** [reply]
Thanks for the quick reply! I guess I'll install the older versions then. I really want to play with that feature :D 
I'm pretty new to using Audacity and AI, but it looks very promising. Keep up the good work!

**Reply by RyanMetcalfeInt8** [reply]
Sure thing! Music Continuation is pretty fun to play with too. Not exactly the same as remix, but it's fun to give it 10s snippets of audio, and see what it comes up with :smile:

https://github.com/intel/openvino-plugins-ai-audacity/blob/main/doc/feature_doc/music_generation/README.md#audio-continuation-on-new-track

**Reply by ScanderDevMode** [reply]
@ViktorKuzev 
You can now use it with Audacity 3.6.4. The compatibility is updated.
Download from this link - https://github.com/intel/openvino-plugins-ai-audacity/releases
